examples/stable_diffusion_v2/inference/sd_infer.py

```diff
@@ -42,7 +42,6 @@ def main(args):
         config,
         ckpt=config.model.pretrained_ckpt,
         use_lora=args.use_lora,
-        lora_rank=args.lora_rank,
```

Comment:
Why deleting `lora_rank` here?

Reply:
No need. we infer rank_rank from checkpoint.